src/components/Modals/AddContactModal.jsx

```diff
@@ -20,9 +20,9 @@ import useNotification from '@hooks/useNotification';
 import { FormSection } from '../Form';
 
 /**
- * @memberof Modals
+ * @memberof Contcts
  * @name renderWebId
- * @param {string} username - Username to convert into a webId
+ * @param {string} username - username to convert into a webId
```

Comment:
I don't think the correction is needed here.

Reply:
Alright, think we can just leave this for now. There are other JSDocs that still uses lowercasing, so I think that's fine for now